pyccel/stdlib/ndarrays/ndarrays.c

```diff
@@ -40,7 +40,7 @@ t_ndarray   array_create(int32_t nd, int32_t *shape, enum e_types type)
             arr.type_size = sizeof(double complex);
             break;
     }
-    arr.is_slice = false;
+    arr.is_view = false;

```

Comment:
I don't see any code that sets `is_view = true`. Are we missing a function that would be called from `_print_AliasAssign`?

Reply:
i am setting it to True in the func `t_ndarray array_slicing(t_ndarray p, ...)`  line 200

Reply:
Ah you're right. Sorry!